NetProbe/Model/NetworkMeasurement.mm

```diff
@@ -120,24 +118,25 @@ -(id) init {
 -(void) run {
     setup_idempotent();
     mk::ooni::HttpInvalidRequestLineTest()
-    .set_backend("http://www.google.com/")
-    .set_verbose()
-    .on_log([self](const char *s) {
-        // XXX OK to send messages to object from another thread?
-        NSString *current = [NSString stringWithFormat:@"%@: %@", [super getDate],
-                             [NSString stringWithUTF8String:s]];
-        NSLog(@"%s", s);
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [self.logLines addObject:current];
-        });
-    })
-    .run([self]() {
-        NSLog(@"http_invalid_request_line testEnded");
-        dispatch_async(dispatch_get_main_queue(), ^{
-            self.finished = TRUE;
-            [[NSNotificationCenter defaultCenter] postNotificationName:@"refreshTable" object:nil];
+        .set_options("backend", "http://213.138.109.232/")
+        .set_options("dns/nameserver", get_dns_server())
```

Comment:
I've added an explicit dns nameserver for http-invalid-request-line and tcp-connect. Not sure whether one is also needed for dns-injection, but perhaps yes. Other options that would be needed are the ones for setting the report file path and the location of the two databases used to geolocate.